pkg/dataplane/test/sync_test.go

```diff
@@ -1137,6 +1139,49 @@ func (suite *syncStreamTestSuite) TestStream() {
 
 	response.Release()
 
+	getItemsInput := v3io.GetItemsInput{
+		Path:               streamPath,
+		ReturnData:         "true",
+		AllowObjectScatter: "true",
+		AttributeNames:     []string{"**"},
+	}
+
+	suite.populateDataPlaneInput(&getItemsInput.DataPlaneInput)
+
+	cursor, err := v3io.NewItemsCursor(suite.container, &getItemsInput)
+	suite.Require().NoError(err, "Failed to get items")
+
+	cursorItems, err := cursor.AllSync()
+	suite.Require().NoError(err)
+
+	// internal use case test
+	for _, cursorItem := range cursorItems {
+		shardName, err := cursorItem.GetFieldString("__name")
+		suite.Require().NoError(err, "Failed to get item name")
+		chunkId, streamData, chunkMetadata, currentChunkMetadata, err := cursorItem.GetShard()
+		suite.Require().NoError(err, "Failed to get stream")
+
+		suite.Require().Equal(0, chunkId, "chunk indexes doesn't match")
+		suite.Require().True(chunkMetadata.ChunkSeqNumber == 0)
+		suite.Require().True(chunkMetadata.FirstRecordSeqNumber == 1)
+		suite.Require().True(chunkMetadata.LengthInBytes == 0)
+		switch shardName {
+		case "0":
+			suite.Require().True(currentChunkMetadata.NextRecordSeqNumber == 2)
+			suite.Require().True(currentChunkMetadata.CurrentChunkLengthBytes == 56)
```

Comment:
It's better to compare these again the test data in `records` instead of using "magic" numbers like 56
```
suite.Require().True(currentChunkMetadata.CurrentChunkLengthBytes == len(records[5].Data))
suite.Require().True(strings.Contains(string(*streamData[0].Data), records[5].Data))
```
This way if the test data changes there won't be a need to update the validations

BTW, why the condition is `strings.Contains`, shouldn't it be `Equal`?

Reply:
validation changes added to pr comments commit.
the data from GetItems isn't a straight forward record data, it includes some extra data,
that is why it is Contains and not Equal.